clippy_lints/src/utils/inspector.rs

```diff
@@ -0,0 +1,478 @@
+#![allow(print_stdout, use_debug)]
+
+//! checks for attributes
+
+use rustc::lint::*;
+use rustc::hir;
+use syntax::ast::{Attribute, MetaItemKind};
+
+/// **What it does:** Dumps every ast/hir node which has the `#[clippy_dump]` attribute
+///
+/// **Why is this bad?** 😈
```

Comment:
not sure what else to do here? remove that question and deviate from the usual doc pattern?